firstvoices/backend/views/search/base_search_views.py

```diff
@@ -70,7 +81,7 @@ def list(self, request, **kwargs):
 
         # Get search results
         try:
-            response = search_query.execute()
+            response = search_query[0:ES_MAX_RESULTS].execute()
```

Comment:
This looks like we will always be retrieving the first 100 search results. Is that right? I would expect that for page 2 with the default page size, we'd be retrieving more like [26-50] here